docs/quickstart.rst

```diff
@@ -34,6 +34,12 @@ configuration.
 Whenever an instance is deleted (through the admin or code), the associated files are
 purged as well.
 
+You can now also generate a Certificate Signing Request (CSR) and a private key in the admin interface.
+
+Generated files are stored at your configured ``PRIVATE_ROOT`` directory.
+
+You can also download the CSR from the admin interface.
```

Comment:
drop the "you can now" - documentation itself is also versioned should it should just describe the $current situation :)